recipes/libsigcpp/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set major_minor = "2.6" %}
+{% set version = major_minor + ".2" %}
+
+package:
+  name: libsigcpp
+  version: {{ version }}
+
+source:
+  fn: libsigc++-{{ version }}.tar.xz
+  url: http://ftp.acc.umu.se/pub/GNOME/sources/libsigc++/{{ major_minor }}/libsigc++-{{ version }}.tar.xz
+  sha256: fdace7134c31de792c17570f9049ca0657909b28c4c70ec4882f91a03de54437
+
+build:
+  number: 0
+    skip: true  # [win]
+
+#requirements:
+#  build:
+#  run:
+
+#test:
+#  imports:
+#  commands:
```

Comment:
The linter will complain if there is no tests. You can do a "exist lib test" like [this one](https://github.com/conda-forge/hdfeos2-feedstock/blob/master/recipe/meta.yaml#L26).